__tests__/routes/web/geographySearch.spec.ts

```diff
@@ -0,0 +1,239 @@
+/**
+ * @jest-environment jsdom
+ */
+
+'use strict';
+
+import { Server, ServerInjectResponse } from '@hapi/hapi';
+
+import { initializeServer } from '../../../src/infrastructure/server';
+import { webRoutePaths } from '../../../src/utils/constants';
+
+describe('Guided Search - Geography Questionnaire Screen', () => {
+  let server: Server;
+  let response: ServerInjectResponse<object>;
+  let document: Document;
+
+  beforeAll((done) => {
+    initializeServer().then(async (s: Server) => {
+      server = s;
+
+      const options = {
+        method: 'GET',
+        url: webRoutePaths.geographySearch,
+      };
+
+      response = await server.inject(options);
+      const rawHTML = response.payload;
+      const parser = new DOMParser();
+      document = parser.parseFromString(rawHTML, 'text/html');
+      done();
+    });
+  });
```

Comment:
can we also test the POST method of this form?

Reply:
Now the scenarios related to the POST route has been added